### ISSUE TITLE
Error: Can't connect to WebDriver. Error: connect ECONNREFUSED 127.0.0.1:443  Please make sure Selenium Server (ChromeDriver or PhantomJS) is running and accessible

### ISSUE THREAD

**Issue by danieltisdale** [Codeception/CodeceptJS#1548]
#### What are you trying to achieve?
- I am trying to upgrade codeceptjs from version 1.4.3 to 2.0.7 and it is causing all of the tests to not be able to connect to webdriver

#### What do you get instead?

> Provide console output if related. Use `--verbose` mode for more details.

```bash
Error: connect ECONNREFUSED 127.0.0.1:443

Please make sure Selenium Server (ChromeDriver or PhantomJS) is running and accessible
    Emitted | test.failed ([object Object])
  ✖ "before each" hook: codeceptjs.before for "User can login to their account and access the account details page" in 4772ms
    [1] Error | Error: Can't connect to WebDriver.
Error: connect ECONNREFUSED 127.0.0.1:443

Please make sure Selenium Server (ChromeDriver or PhantomJS) is running and accessible
Error: Can't connect to WebDriver.
Error: connect ECONNREFUSED 127.0.0.1:443

Please make sure Selenium Server (ChromeDriver or PhantomJS) is running and accessible
    Emitted | test.after ([object Object])
    Emitted | suite.after ([object Object])


```

> Provide test source code if related

```js
Scenario(
  'User can login to their account and access the account details page',
  (I, HomePage) => {
    I.amOnPage('/'); // opens main page of website
    I.click(HomePage.buttons.login);
    I.waitForText('LOG IN', 10);
    I.userLogin();
    HomePage.goToAccountPage();
  },
);
```

### Details

* CodeceptJS version: 2.0.7
* NodeJS Version: 10.15.3
* Operating System: macOSX
* Protractor || WebDriverIO || Nightmare version (if related): webdriverio 5.7.1
* Configuration file:

```js
# paste suite config here
```

**Comment by Vorobeyko** [comment]
please paste codeceptjs config

**Comment by danieltisdale** [comment]
```const { resolve } = require('path');

const getConfig = require('./end-to-end/buildConfig');

if (!process.env.SAUCE_ACCESS_KEY || !process.env.SAUCE_USERNAME) {
  require('dotenv').config({
    path: resolve('./config/.saucelabs.env'),
  });
}

if (!process.env.LOGIN_WALL_PASSWORD) {
  require('dotenv').config({
    path: resolve('./config/.env'),
  });
}

const {
  SITE_URL,
  BUILD_ID,
  IS_LOCAL,
  TEST_TAG,
  CIRCLE_LINK,
  GITHUB_LINK,
} = getConfig(process.env);

// eslint-disable-next-line no-console
console.log('Testing against:', SITE_URL);

exports.config = {
  tests: './end-to-end/tests/*.js',
  timeout: 100000,
  output: './end-to-end/output',
  plugins: {
    retryFailedStep: {
      enabled: true,
    },
  },
  helpers: {
    WebDriver: {
      desiredCapabilities: {
        browserName: 'chrome',
        version: 'latest',
        platform: 'Windows 10',
        screenResolution: '1920x1080',
      },
      url: SITE_URL,
      smartWait: 5000,
      browser: 'chrome',
      windowSize: '1920x1080',
      services: ['sauce'],
      user: process.env.SAUCE_USERNAME,
      key: process.env.SAUCE_ACCESS_KEY,
      host: 'ondemand.saucelabs.com',
      port: 80,
    },
    SauceLabs: {
      require: './end-to-end/helpers/SauceLabs.js',
      BUILD_ID,
      IS_LOCAL,
      TEST_TAG,
      CIRCLE_LINK,
      GITHUB_LINK,
    },
  },
  multiple: {
    chrome: {
      chunks: 4,
      browsers: ['chrome'],
    },
    firefox: {
      chunks: 4,
      browsers: ['firefox'],
    },
  },
  include: {
    I: './end-to-end/steps.js',
    PropertyDetailsPage: './end-to-end/pages/propertyDetailsPage.js',
    SearchPage: './end-to-end/pages/searchPage.js',
    HomePage: './end-to-end/pages/homePage.js',
    AccountPage: './end-to-end/pages/accountPage.js',
    AgentsPage: './end-to-end/pages/agentsPage.js',
    AgentResultsPage: './end-to-end/pages/agentSearchPage.js',
    ContactPage: './end-to-end/pages/contactPage.js',
    BuyPage: './end-to-end/pages/buyPage.js',
    MortgagePage: './end-to-end/pages/mortgagePage.js',
    SellPage: './end-to-end/pages/SellPage.js',
    CareersPage: './end-to-end/pages/careersPage.js',
  },
  bootstrap: false,
  name: 'Merlin',
};
```

**Comment by LukoyanovE** [comment]
I had a similar issue with browserstack. Problem was with "hostname" overwrite.
I made a fix for it #1506 , released in 2.0.6.
So it should work correctly.

Maybe, you have `host` rewrite to undefined or empty string somewhere ?
Or not updated to 2.0.6.

**Comment by danieltisdale** [comment]
@LukoyanovE I have been unable to get this to work through both Sauce Labs and localhost. Any ideas on what might be causing this?

**Comment by VikalpP** [comment]
Fixed in CodeceptJS v2.1.5.
Check out PR at #1723.
